avocado/core/plugin_interfaces.py

```diff
@@ -213,7 +213,14 @@ class VarianterPlugin(Plugin):
     @abc.abstractmethod
     def __iter__(self):
         """
-        Yield variant in form of (variant_id, (variant, mux_path))
+        Yields all variants
```

Comment:
As this was introduced by a commit in the same PR, docstring such as this one should be added alongside the original commit.  In this specific case, it should then have been updated in the previous commit, when the format of the yelded variant changes to a dict.